test/unit/functional/collapse-and-batch.spec.js

```diff
@@ -199,25 +199,23 @@ describe('Collapse and Batch', function() {
             subscribe(noOp, done, done);
     });
 
+    // HACK: This test is not passing
+    // NOTE: I made the test easier to read by simplifying it. I'm still not sure what it's testing.
     it('should validate batching/collapsing makes two request since its onNextd without toArray().', function(done) {
         var serviceCalls = 0;
         var routes = [{
             route: 'lists[{keys:ids}]',
             get: function(aliasMap) {
-                return Observable.
-                    from(aliasMap.ids).
-                    map(function(id) {
-                        if (id === 0) {
-                            return {
-                                path: ['lists', id],
-                                value: $ref('two.be[956]')
-                            };
-                        }
-                        return {
-                            path: ['lists', id],
-                            value: $ref('lists[0]')
-                        };
-                    });
+                return Observable.of(
```

Comment:
Yeah, @jhusain and I had gone over this and confirmed that it was never going to be any different, but I see what you're saying. I'll change it back.